pages/documentation/migration-V3.md

```diff
@@ -0,0 +1,822 @@
+        <strong>Was: </strong>white 
+      </li>
+      <li>
+        <strong>Updated: </strong>$theme-alert-icon-size: 5 <br>
+        <strong>Was: </strong>4
+      </li>
+    </ul>
+    <p><strong>What to do:</strong></p>
+    <ol>
+      <li>Search your codebase for any instances of <strong>the old theme</strong></li>
+      <li>Replace it with <strong>the new setting</strong></li>
+    </ol>
+    <p><strong><span class="usa-tag bg-accent-cool-darker">Markup</span> We updated usa-footer__logo-heading to use a p instead of an h3.</strong> We improved the accessibility of the footer by converting a non-semantic heading into paragraph text.</p>
+    <p><strong>What to do: </strong></p>
+    <ol>
+      <li>TK</li>
+      <li>TK</li>
+    </ol>
+  </div>
+</div>
+
+### 3. Install the USWDS 3.0 package
+
+{:.border-top-2px.border-base-lighter.padding-top-1}
+1. In a terminal window, navigate to your project root. 
+2. Install USWDS 3.0 with **npm install @uswds/uswds --save --save-exact**
+3. Uninstall USWDS 2 with **npm uninstall uswds**
+
+### 4. Update your Sass compiler settings and recompile CSS
+
+{:.border-top-2px.border-base-lighter.padding-top-1}
+The location of the USWDS source files is different in USWDS 3.0. You'll need to update your compiler settings to compile your Sass from USWDS 3.0.
+
+<div class="usa-accordion usa-accordion--bordered">
+  <!-- Use the accurate heading level to maintain the document outline -->
+  <h4 class="usa-accordion__heading">
+    <button
+      class="usa-accordion__button"
+      aria-controls="m-a6"
+    >
+      If you're using USWDS Gulp
+    </button>
+  </h4>
+  <div id="m-a6" class="usa-accordion__content usa-prose">
+    <ol>
+      <li>Search for a line like <strong>const USWDS = "node_modules/uswds/dist"</strong> in your Gulp setup. This indicates that you're using the Gulp setup we distributed as USWDS Gulp.</li>
+      <li>
+        Update the USWDS <strong>const</strong> to the updated uswds package location
+        <p><strong>Old</strong></p>
+          <code>const USWDS = "node_modules/uswds/dist";</code>
+        <p><strong>New</strong></p>
+          <code>const USWDS = "node_modules/@uswds/uswds";</code>
+          <br>
+      </li>
+      <li> 
+        Search for <strong>includePaths</strong> in your project's Gulp files. The paths in this list are where the Sass compiler looks for your source files. In USWDS 3.0
+        <p><strong>Old</strong></p>
+        <pre><code>.pipe(
+  sass({
+    includePaths: [
+      PROJECT_SASS_SRC,
+      `${USWDS}/scss`,
+      `${USWDS}/scss/packages`,
+    ],
+  })
+)</code></pre>
+        <p><strong>New</strong></p>
+        <pre><code>.pipe(
+  sass({
+    includePaths: [
+      PROJECT_SASS_SRC,
+      `${USWDS},
+      `${USWDS}/packages`,
+    ],
+  })
+)</code></pre>
+      </li>
+      <li>Recompile your Sass as usual. When it compiles, it is now using USWDS 3.0!</li>
+    </ol>
```

Comment:
Expanding on my earlier comment first made [here](https://github.com/uswds/uswds-site/pull/1529/files/6a3e00ee24c290f07d9409a7f646e1c91acc0f34#r856357821).

I did a fresh install of USWDS Gulp on a test project and found the instructions didn't match the current standard USWDS Gulp references. Working on the assumption that many users will be using the defaults from USWDS Gulp, I updated the instructions in this section. 

@mejiaj can you confirm that the steps look good to you? 
Is the +/- diff a clear way to communicate the changes? 

Snapshot from federalist:
![image](https://user-images.githubusercontent.com/93996430/165133846-504ca994-2182-4cdd-acae-80b533408005.png)

:warning: I couldn't add suggested code in this comment because it struggled with the nested code blocks, but I included the the proposed update code below for reference. If this generally looks good from the dev side, I can push up my local branch for copyediting.  :warning: 
1. Search for a line like `const uswds = require("./node_modules/uswds-gulp/config/uswds");` or `const uswds = "node_modules/uswds/dist"` in your Gulp setup. This indicates that you're using the Gulp setup we distributed as USWDS Gulp.
1. Update the USWDS `const` elements to the updated USWDS package location:
  ```diff
  - const pkg = require("./node_modules/uswds/package.json");
  + const pkg = require("./node_modules/@uswds/uswds/package.json"); 
  ```
  ```diff
  - const uswds = require("./node_modules/uswds-gulp/config/uswds"); 
  + const uswds = "node_modules/@uswds/uswds";
  ``` 
1. Search for references to `${uswds}` in the `includePaths` and `gulp.src()` found in your project’s Gulp files. These paths tell the Sass compiler where to look for USWDS source files. 
   
   Some of our file directories have moved in USWDS 3.0, and it is necessary to point Gulp to the correct location inside `node_modules/@uswds/uswds`. Below are snippets from the standard USWDS Gulp references and their necessary updates:
   ```diff
   ...
   - gulp.src(`${uswds}/scss/theme/**/**`)
   + gulp.src(`${uswds}/dist/theme/**/**`)
   ...
   - gulp.src(`${uswds}/fonts/**/**`)
   + gulp.src(`${uswds}/dist/fonts/**/**`)
   ...
   - gulp.src(`${uswds}/img/**/**`)
   + gulp.src(`${uswds}/dist/img/**/**`)
   ...
   - gulp.src(`${uswds}/js/**/**`)
   + gulp.src(`${uswds}/dist/js/**/**`)
   ...
     includePaths: [
      PROJECT_SASS_SRC,
   -     `${uswds}/scss`,
   +     `${uswds}/dist/scss`,
   -     `${uswds}/scss/packages`,
   +     `${uswds}/packages`,
     ],
   ```
1. Recompile your Sass as usual and check that your files compiled as expected. When it compiles successfully, it is now using USWDS 3.0!

Reply:
Done!